alphabase/psm_reader/keys.py

```diff
@@ -0,0 +1,71 @@
+class ConstantsClass(type):
+    """A metaclass for classes that should only contain string constants."""
+
+    def __setattr__(self, name, value):
+        raise TypeError("Constants class cannot be modified")
+
+    def get_values(cls):
+        """Get all user-defined string values of the class."""
+        return [
+            value
+            for key, value in cls.__dict__.items()
+            if not key.startswith("__") and isinstance(value, str)
+        ]
+
+
+class PsmDfCols(metaclass=ConstantsClass):
```

Comment:
Shall we make this const like`PSM_DF_COLS`?

Reply:
I think a class is not a bad idea. In alphadia I want to have a superset of this. So I could inherit and extend it.

Reply:
if it's only about the naming: I'd prefer to stick to the current name, first not to violoate the convention, second because `PSM_DF_COLS.PRECURSOR_MZ` is imho harder to read than `PsmDfCols.PRECURSOR_MZ` (personal preference)

Reply:
OK!